src/Controller/Host/CreateEventController.php

```diff
@@ -0,0 +1,36 @@
+<?php
+
+namespace App\Controller\Host;
+
+use App\Entity\Experience;
+use App\Repository\EventRepository;
+use App\Request\EventRequest;
+use App\Service\EventService;
+use Symfony\Bundle\FrameworkBundle\Controller\AbstractController;
+use Symfony\Component\HttpFoundation\JsonResponse;
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\HttpFoundation\Response;
+use Symfony\Component\Routing\Annotation\Route;
+
+class CreateEventController extends AbstractController
```

Comment:
Please rename controller to `EventController` and put verbs in method names such as `create`